packages/component-library/src/SingleToggleGroup/index.tsx

```diff
@@ -63,6 +66,7 @@ export const SingleToggleGroup = ({
           )}
           data-size="sm"
           data-variant="ghost"
+          data-rounded={rounded ? true : undefined}
```

Comment:
This should be `rounded ? "" : undefined` (the idea is that we want data attributes that are boolean to either be present or not, rather than have the value `true`, to be consistent with how react-aria does things)